app/main/auto-updater/auto-updater.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Auto Updater
+ * 
+ * Similar to https://electronjs.org/docs/api/auto-updater#events
+ * See https://electronjs.org/docs/tutorial/updates for documentation
+ */
+import { app, autoUpdater, dialog } from 'electron';
+import moment from 'moment';
+import B from 'bluebird';
+import { getUpdate } from './update-checker';
+import { getFeedUrl } from './config';
+  
+autoUpdater.setFeedURL(getFeedUrl(app.getVersion()));
+
+/** 
+ * Check for new updates
+ */
+export async function checkNewUpdates (fromMenu) {
+  // autoupdate.checkForUpdates always downloads updates immediately 
+  // This method (getUpdate) let's us take a peek to see if there is an update 
+  // available before calling .checkForUpdates
+  const update = await getUpdate(app.getVersion());
+  if (update) {
+    let {name, notes, pub_date} = update;
+    pub_date = moment(pub_date).format('MMM Do YYYY, h:mma');
```

Comment:
even if `update` really has to have snake_case, we should name our own variables using camelCase